src/panels/lovelace/hui-unused-entities.ts

```diff
@@ -42,14 +55,56 @@ export class HuiUnusedEntities extends LitElement {
     this._createElements();
   }
 
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    const lovelace = this.lovelace!;
+
+    let editModeChanged = false;
+
+    if (changedProperties.has("lovelace")) {
+      const oldLovelace = changedProperties.get("lovelace") as Lovelace;
+      editModeChanged =
+        !oldLovelace || lovelace.editMode !== oldLovelace.editMode;
+    }
+
+    if (editModeChanged) {
+      this._createElements();
+    }
+  }
+
   protected render(): TemplateResult | void {
-    if (!this._config || !this._hass) {
+    if (!this._config || !this._hass || !this.lovelace) {
       return html``;
     }
 
     return html`
       ${this.renderStyle()}
-      <div id="root">${this._elements}</div>
+      <div id="root">
+        ${this.lovelace.editMode
```

Comment:
We should no longer allow opening unused entities from the main menu, it should only be available in edit mode.